projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loader.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, ViewEncapsulation, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+  selector: 'kal-loader',
+  template: `
+    <p> Chargement en cours </p>
+  `,
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class KalLoaderComponent implements OnInit {
```

Comment:
Remove onInit

Reply:
We should be able to change to loading message